DevOps Engineer/seo-dev-ops-engineer

```diff
@@ -0,0 +1,48 @@
+Job description
```

Comment:
Needs to follow the standard Markdown formatting for a job description - see Senior DevOps Engineer, but e.g.

```
# DevOps Engineer
Grade: SEO

Body content...

## You will:

- Bullet list item 1
- Item 2
```

Reply:
Hi Luke, I copy and pasted the text out of our actual advert, which has been through the usual checks by HR etc. around wording. Also, its really hard to unpick the references to TS as they are embedded in the JD really. I’m happy to change it to add in the .md markers for example to make it look neater but at the end of the day the JD’s are in text anyway when they are sent to HR. Isn’t the idea to have something that people can copy and change - rather than a generic example that they use in .md (as we don't use .md for adverts)

Reply:
Thanks Andy. Probably just replace any TS references with a [division] placeholder? (as per next comment)

Yes, agree on purpose, and people often cut & paste from the rendered page (rather than use markdown). 

We're moving this whole site to a different template, which will allow for a document version to be attached to each page - that should make it easier for all to use.

Reply:
ok